### PR TITLE
Remove/replace references to firecloud

### PR DESCRIPTION
**Pull request by NatalieEO**
dockstore/dockstore#2456

### CODE REVIEW

File: docs/_static/language-support.html [hunk 1]
```diff
@@ -68,13 +68,12 @@
             <td style="padding: 5px;">Launch-with Platforms</td>
             <td style="background-color:#fcf8e3; padding: 5px;">Not yet!</td>
             <td style="background-color:#dff0d8; padding: 5px;">
-                <a href="/firecloud-launch-with/">FireCloud</a> (workflows only)<sup>[2]</sup>
+                <a href="../end-user-topics/terra-launch-with.html">Terra</a> (workflows only)<sup>[2]</sup>
                 <br>
-                <a href="/dnastack-launch-with/">DNAstack</a> (workflows only)<sup>[3]</sup>
+                <a href="../end-user-topics/dnastack-launch-with/">DNAstack</a> (workflows only)<sup>[3]</sup>
```
Comment (denis-yuen) [inline docs/_static/language-support.html]: Are dnastack and dnanexus changing locations or was this previously incorrect?

Comment (NatalieEO) [reply docs/_static/language-support.html]: Previously incorrect. But your comment made me realize I forgot to add the .html at the end of the other two so they were still broken anyway.

Comment (garyluu) [reply docs/_static/language-support.html]: Missing link-checker test?

Comment (denis-yuen) [reply docs/_static/language-support.html]: > Missing link-checker test?

Elaborate? 
If does seem odd that the link checker didn't catch this in the old repo. 
But perhaps this section has changed going to readthedocs and I don't know if readthedocs has a link checker setup yet @agduncan94

Comment (garyluu) [reply docs/_static/language-support.html]: This repo is missing CI entirely.

Comment (garyluu) [reply docs/_static/language-support.html]: Looks like https://docs.dockstore.org/docs/user-tutorials/language-support/ currently works

Comment (garyluu) [reply docs/_static/language-support.html]:
```
- docs/_static/language-support.html
  *  internally linking to /advanced-features/#file-provisioning, which does not exist (line 104)
     <a href="/advanced-features/#file-provisioning">File Provisioning</a>
  *  internally linking to /advanced-features/#notifications, which does not exist (line 159)
     <a href="/advanced-features/#notifications">Notifications</a>
  *  internally linking to /best-practices/#authorship-metadata, which does not exist (line 83)
     <a href="/best-practices/#authorship-metadata">Yes</a>
  *  internally linking to /dnanexus-launch-with/, which does not exist (line 75)
     <a href="/dnanexus-launch-with/">DNAnexus</a>
  *  internally linking to /dnastack-launch-with/, which does not exist (line 73)
     <a href="/dnastack-launch-with/">DNAstack</a>
  *  internally linking to /firecloud-launch-with/, which does not exist (line 71)
     <a href="/firecloud-launch-with/">FireCloud</a>
  *  internally linking to /getting-started-with-dockstore/, which does not exist (line 34)
     <a href="/getting-started-with-dockstore/">Tool registration</a>
  *  internally linking to /hosted-tools-and-workflows/, which does not exist (line 46)
     <a href="/hosted-tools-and-workflows/">Hosted Workflows</a>
  *  internally linking to /launch/#dockstore-cli, which does not exist (line 94)
     <a href="/launch/#dockstore-cli">Local workflow engines</a>
  *  internally linking to /nfl-best-practices/#authorship-metadata, which does not exist (line 85)
     <a href="/nfl-best-practices/#authorship-metadata">Yes</a>
  *  internally linking to /wdl-best-practices/#authorship-metadata, which does not exist (line 84)
     <a href="/wdl-best-practices/#authorship-metadata">Yes (wdl version&lt;=draft-2)</a>
  *  internally linking to /workflows/, which does not exist (line 40)
     <a href="/workflows/">Workflow registration</a>
  *  trying to find hash of /advanced-features/#file-provisioning, but /home/gluu/dockstore-documentation/docs/advanced-features does not exist (line 104)
     <a href="/advanced-features/#file-provisioning">File Provisioning</a>
  *  trying to find hash of /advanced-features/#notifications, but /home/gluu/dockstore-documentation/docs/advanced-features does not exist (line 159)
     <a href="/advanced-features/#notifications">Notifications</a>
  *  trying to find hash of /best-practices/#authorship-metadata, but /home/gluu/dockstore-documentation/docs/best-practices does not exist (line 83)
     <a href="/best-practices/#authorship-metadata">Yes</a>
  *  trying to find hash of /launch/#dockstore-cli, but /home/gluu/dockstore-documentation/docs/launch does not exist (line 94)
     <a href="/launch/#dockstore-cli">Local workflow engines</a>
  *  trying to find hash of /nfl-best-practices/#authorship-metadata, but /home/gluu/dockstore-documentation/docs/nfl-best-practices does not exist (line 85)
     <a href="/nfl-best-practices/#authorship-metadata">Yes</a>
  *  trying to find hash of /wdl-best-practices/#authorship-metadata, but /home/gluu/dockstore-documentation/docs/wdl-best-practices does not exist (line 84)
     <a href="/wdl-best-practices/#authorship-metadata">Yes (wdl version&lt;=draft-2)</a>
htmlproofer 3.7.6 | Error:  HTML-Proofer found 18 failures!
```

**Pull request merged**